openshift/buildConfig-template.yaml

```diff
@@ -28,38 +28,28 @@ objects:
         name: "kebechet-job:${IMAGE_STREAM_TAG}"
     runPolicy: Serial
     source:
-      git:
-        uri: ${GITHUB_URL}
-        ref: ${GITHUB_REF}
       type: Git
+      git:
+        uri: "https://github.com/harshad16/kebechet.git"
+        ref: "feature/dockerfile"
```

Comment:
OK for now, but we later we will keep URI and ref as parameters.

Reply:
lets do it now, its an easy copy from some of the user-api buildconfig template.

Reply:
No need, its already present in the original buildconfig. But yes - let's do it now, @harshad16 keep parameters as they were. We can merge this and continue with templates for cronjob and configmap.

Reply:
sure